### PR TITLE
Upload preview for SVGs

### LINKED ISSUE #5436: [v4] svg preview in file upload & file replace not working
## Description
When uploading and replacing files, no preview gets loaded for svg files.

**Expected behavior**  
I'd expect to be able to preview the svg file.

**Screenshots**  
Screenshot 2023-08-01 at 11 04 16

Screenshot 2023-08-01 at 11 00 34

**Kirby Version**  
4.0.0-alpha.5

### PR DESCRIPTION
**Pull request by distantnative**
### Fixes
- Upload dialog: preview for other file types than images

### Enhancements
- Upload dialog: preview SVG files
#5436

### CODE REVIEW

File: panel/src/components/Dialogs/UploadDialog.vue [hunk 1]
```diff
@@ -28,6 +28,11 @@
 								:src="file.url"
 								back="pattern"
 							/>
+							<k-image-frame
+								v-else-if="file.type.match('svg')"
```
Comment (lukasbestle) [inline panel/src/components/Dialogs/UploadDialog.vue]: Would this also match a file whose extension only *contains* `svg` (e.g. `svgz` or `abcdesvgabcde`)? Same for the pixel formats above.

Comment (distantnative) [reply panel/src/components/Dialogs/UploadDialog.vue]: Good point. I think we need to differentiate between upload and upload replace. The latter matches on the url. There it is on the extension. This one is on the mime type. I think both could have problems as you point out.

Comment (distantnative) [reply panel/src/components/Dialogs/UploadDialog.vue]: Probably best to switch to using mime types for both dialogs. And then look for exact matches on the full mime type strings.

Comment (distantnative) [reply panel/src/components/Dialogs/UploadDialog.vue:32]:
```html
<k-image-frame
	v-if="
		[
			'image/jpeg',
			'image/jpg',
			'image/gif',
			'image/png',
			'image/webp',
			'image/avif'
		].includes(file.type)
	"
	:cover="true"
	:src="file.url"
	back="pattern"
/>
<k-image-frame
	v-else-if="file.type === 'image/svg+xml'"
	:src="file.url"
	back="pattern"
/>
```

Comment (distantnative) [reply panel/src/components/Dialogs/UploadDialog.vue]: https://github.com/getkirby/kirby/pull/5461/commits/8760ce4236c68595d3f418a96c03a00d8c09974e

**Review by afbora — APPROVED** [review]
LGTM

**Pull request merged**